<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AstraExplorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>781</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
   </widget>
   <widget class="QTreeWidget" name="dbObjects">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>251</width>
      <height>751</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Keyspaces</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableData">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>310</y>
      <width>781</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>test</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Table Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Table Definition</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>exitMenuAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbObjects</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>ItemChangedAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>exitMenuSignal()</signal>
  <signal>itemChangedSignal()</signal>
  <slot>exitMenuAction()</slot>
  <slot>ItemChangedAction()</slot>
 </slots>
</ui>
